### DISCUSSION TITLE
ui5-webcomponents  DateTimePicker how to change language ??

### DISCUSSION THREAD

**Discussion by pupss89345** [UI5/webcomponents#10020] [Q&A] [answered]
I found react version DateTimePicker can easy change language  
https://github.com/SAP/ui5-webcomponents/issues/6223
![image](https://github.com/user-attachments/assets/58f6e34c-fe79-4339-b9bf-df54e095974f)
but I don't know how to change ui5-webcomponents version DateTimePicker language I only found old post 
https://github.com/SAP/ui5-webcomponents/issues/5356
I refer https://sap.github.io/ui5-webcomponents/docs/advanced/using-i18n-for-apps/ ,but not sure how to apply on DateTimePicker

**Comment by petyabegovska** [comment, marked as answer]
Hi, the documentation is here: https://sap.github.io/ui5-webcomponents/docs/advanced/using-assets/

For example:

import "@ui5/webcomponents/dist/Assets.js";

and use:

<ui5-date-picker></ui5-date-picker>

with another language and another theme:

import { setLanguage } from "@ui5/webcomponents-base/dist/config/Language.js";
import { setTheme } from "@ui5/webcomponents-base/dist/config/Theme.js";

setLanguage("es");
setTheme("sap_fiori_3_hcb");

The ui5-date-picker component will have all translatable texts in Spanish, and the Spanish format settings (e.g. date format) will be used, and will be rendered with the sap_fiori_3_hcb accessibility theme instead of the default theme.

However, it doesn't work in the latest code and we need to investigate further. I will keep you updated.

Can you let me know which version of UI5 Web Components are you using?

Best regards,
Petya

**Reply by pupss89345** [reply]
OH !!! Thanks you a lot ~ 
It's worked for me ~
I'm use version below 
![image](https://github.com/user-attachments/assets/3fa80c4e-51b4-47d7-ae4c-c9fa4846bae4)
